### PR TITLE
Don't show the 'preview' banner on published gallery

### PR DESCRIPTION
**Pull request by xster**
b/77981835

### CODE REVIEW

**Comment by tvolkert** [comment]
Mutating source files hacky.

What if instead we had a `main_no_banner.dart`, and the shell script called build with `-t lib/main_no_banner.dart,`?

**Comment by xster** [comment]
I was hesitant to have our publishing mechanism to be the only integration test that tests `-t` XD. Will change

**Comment by tvolkert** [comment]
Then we should add more integration tests :-P

**Comment by xster** [comment]
PTAL

File: dev/bots/travis_script.sh [hunk 1]
```diff
@@ -10,25 +10,43 @@ if [ "$SHARD" = "build_and_deploy_gallery" ]; then
   if [ "$TRAVIS_OS_NAME" = "linux" ]; then
     echo "Building Flutter Gallery for Android..."
     export ANDROID_HOME=`pwd`/android-sdk
-    (cd examples/flutter_gallery; flutter build apk --release)
+    (
+      cd examples/flutter_gallery;
```
Comment (tvolkert) [inline dev/bots/travis_script.sh]: I think you don't need the semicolons when it's split into multiple lines

File: dev/bots/travis_script.sh [hunk 2]
```diff
@@ -10,25 +10,43 @@ if [ "$SHARD" = "build_and_deploy_gallery" ]; then
...
+      flutter build apk --release -t lib/main_publish.dart
+    )
     echo "Android Flutter Gallery built"
     if [[ "$TRAVIS_PULL_REQUEST" == "false" && "$TRAVIS_BRANCH" == "dev" && $version != *"pre"* ]]; then
       echo "Deploying to Play Store..."
-      (cd examples/flutter_gallery/android; bundle install && bundle exec fastlane deploy_play_store)
+      (
+        cd examples/flutter_gallery/android; \
```
Comment (tvolkert) [inline dev/bots/travis_script.sh]: Similar here - I think this block can just be:

```sh
cd examples/flutter_gallery/android
bundle install
bundle exec fastlane deploy_play_store
```

Comment (tvolkert) [reply dev/bots/travis_script.sh]: same applies below

File: dev/bots/travis_script.sh (hunk 1 above)
Comment (tvolkert) [reply dev/bots/travis_script.sh]: If you edit that example to:

```sh
#!/bin/bash
# GNU bash, version 4.3.46

(
  echo a
  echo b
  echo c 
)
```

Then it does the right thing.  A trailing backslash means "ignore this newline"

**Review by tvolkert — APPROVED** [review]

**Pull request merged**